src/Trinity.Core/Storage/LocalMemoryStorage/LocalMemoryStorage.DiskIO.cs

```diff
@@ -31,7 +31,16 @@ public bool LoadStorage()
         {
             lock (m_lock)
             {
-                if(TrinityErrorCode.E_SUCCESS != CSynchronizeStorageRoot()) { return false; }
+                try
+                {
+                    StorageBeforeLoad();
+                }
+                catch (Exception ex)
+                {
+                    Log.WriteLine(LogLevel.Error, "StorageBeforeLoad event handler: {0}", ex.ToString());

```

Comment:
Could use some revise like: An error occured in the StorageBeforeLoad handler...